website/docs/getting-started/2022-08-05-getting-started.md

```diff
@@ -31,63 +31,63 @@ Once everything is set up, you are ready to go!
 
-![img1.3](img/1.3.png)
+   ![img1.3](img/1.3.png)
 
-4) Click on ‘Build’ and that’s it. You will see each instruction from your Containerfile being executed to build the image. 
+4. Click on ‘Build’ and that’s it. You will see each instruction from your Containerfile being executed to build the image.
 
-![img1.4](img/1.4.png)
+   ![img1.4](img/1.4.png)
 
-5) Once completed, Click on “Done” above to continue. You will see that your image has been added to the Images section of Podman Desktop. 
+5. Once completed, Click on “Done” above to continue. You will see that your image has been added to the Images section of Podman Desktop.
 
-![img1.5](img/1.5.png)
+   ![img1.5](img/1.5.png)
 
 ### 2. Create and Run the Container
 
 Following are the steps to create and start a container from the image.
 
-1) Once the image is built, you can see its name (in our case, ‘my-custom-image’) and other details in the Images section. 
+1. Once the image is built, you can see its name (in our case, ‘my-custom-image’) and other details in the Images section.
 
-![img2.1](img/2.1.png)
+   ![img2.1](img/2.1.png)
 
-2) Simply hover over the name of the image and you will see a list of buttons to the right. In case you are confused about the action of each button, just take the cursor over each button and wait for it to tell the action it does. 
+2. Simply hover over the name of the image and you will see a list of buttons to the right. In case you are confused about the action of each button, just take the cursor over each button and wait for it to tell the action it does.
 
-![img2.2](img/2.2.png)
+   ![img2.2](img/2.2.png)
 
-3) Upon clicking the ‘Run Image’ button, a window will appear and it will ask you to Create the container by giving it a name and mapping the desired port number. 
+3. Upon clicking the ‘Run Image’ button, a window will appear and it will ask you to Create the container by giving it a name and mapping the desired port number.
 
-![img2.3](img/2.3.png)
+   ![img2.3](img/2.3.png)
 
-4) Once you click on “Start Container”, you will see some changes in the Containers section. Upon hovering your cursor over the newly created container, you will see a set of buttons allowing you to work with it.
+4. Once you click on “Start Container”, you will see some changes in the Containers section. Upon hovering your cursor over the newly created container, you will see a set of buttons allowing you to work with it.
 
-![img2.4](img/2.4.png)
+   ![img2.4](img/2.4.png)
 
-5) To check if it is running our application as desired, I will open the URL in the browser (in this case, localhost:8080 because that’s where we mapped our port). This can be done by clicking on the ‘Open Browser’ button.
+5. To check if it is running our application as desired, I will open the URL in the browser (in this case, localhost:8080 because that’s where we mapped our port). This can be done by clicking on the ‘Open Browser’ button.
```

Comment:
```suggestion
5. To check if it is running our application as desired, open the URL in the browser (in this case, localhost:8080 because that’s where we mapped our port). This can be done by clicking on the ‘Open Browser’ button.
```